CodeListLibrary_project/clinicalcode/entity_utils/constants.py

```diff
@@ -78,8 +87,125 @@ class GROUP_PERMISSIONS(int, Enum):
 }
 
 '''
-    [!] Note: Will be moved to a table once tooling is finished, accessible through the 'base_template_version'
+    Entity creation related defaults
+'''
+CREATE_WIZARD_ASIDE = 'components/create/aside.html'
+CREATE_WIZARD_SECTION_START = 'components/create/section/section_start.html'
+CREATE_WIZARD_SECTION_END = 'components/create/section/section_end.html'
+CREATE_WIZARD_INPUT_DIR = 'components/create/inputs'
+
+'''
+    Used to strip userdata from models when JSONifying them
+        e.g. user account, user profile, membership
+'''
+USERDATA_MODELS = [str(User)]
+STRIPPED_FIELDS = ['SearchVectorField']
 
+'''
+    [!] Note: Could be moved once we determine who has responsibility to create/edit FieldTypes.json
+              and where it should be located
+    
+    Used to define:
+        - Hashmap whose value reflects how to display template-related data
+        - By ./create and ./detail pages for field components, how to organise the page, and so forth
+'''
+field_types = {
+    "create_sections": [
+        {
```

Comment:
Awaiting answers regarding where we would like to store this - temporarily held within `constants.py` for development, will eventually be held elsewhere.